source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -20,9 +20,9 @@ database. To retrieve data, use read operations.
 
 Read operations allow you to do the following:
 
-- Retrieve a subset of documents from your collection using a :ref:`find operation <retrieve-find>` 
+- Retrieve a subset of documents from your collection using a :ref:`find operation <retrieve-find>`
 - Perform transformations on retrieved documents from your collection using an :ref:`aggregate operation <retrieve-aggregate>`
-- Monitor real-time changes to your database using a :ref:`change stream <retrieve-watch>` 
+- Monitor real-time changes to your database using :ref:`<retrieve-watch>`
```

Comment:
This text currently reads as: Monitor real-time changes to your database using Open Change Streams. I think we just want it to be "Change Streams" (unless we are intending to call the feature itself "Open Change Streams")

```suggestion
- Monitor real-time changes to your database using :ref:`change streams <retrieve-watch>`
```